src/main/java/dev/hephaestus/glowcase/client/render/block/entity/TextBlockEntityRenderer.java

```diff
@@ -72,15 +72,7 @@ public void renderBaked(TextBlockEntity blockEntity, MatrixStack matrices, Verte
 				matrices.translate(0, 0, 0.025D);
 			}
 
-			if (blockEntity.shadowType == TextBlockEntity.ShadowType.DROP) {
-				// Don't use the vanilla shadow rendering - it breaks when you try to use it in 3D
```

Comment:
Have you tested to make sure it doesn't break anymore?

Reply:
Yes (see discord)

Reply:
For posterity, it would be good to summarize discord discussions to make them visible.

Reply:
In short placed it, rotated it and checked from multiple angles and distances and it looked